Source/Particles/SpeciesPhysicalProperties.H

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2020 Maxence Thevenet
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WARPX_SPECIESPHYSICALPROPERTIES_H_
+#define WARPX_SPECIESPHYSICALPROPERTIES_H_
+
+#include "WarpXConst.H"
+
+#include <AMReX_REAL.H>
+#include <AMReX_AmrCore.H>
+
+#include <map>
+#include <limits>
+
+enum struct PhysicalSpecies{unspecified=0, electron, positron, photon, hydrogen};
+
+static PhysicalSpecies from_string(std::string species)
+{
+    if( species=="unspecified" )
+        return PhysicalSpecies::unspecified;
+    if( species=="electron" )
+        return PhysicalSpecies::electron;
+    if( species=="positron" )
+        return PhysicalSpecies::positron;
+    if( species=="photon" )
+        return PhysicalSpecies::photon;
+    if( species=="hydrogen" )
+        return PhysicalSpecies::hydrogen;
+    amrex::Abort("unknown PhysicalSpecies");
+    return PhysicalSpecies::unspecified;
```

Comment:
This will never be reached, right? You leave this `return` here to get rid of the alert (possible no return)?

Reply:
Correct.